chrome/browser/about_flags.cc

```diff
@@ -1403,12 +1403,14 @@ const FeatureEntry kFeatureEntries[] = {
     {"ui-disable-partial-swap", flag_descriptions::kUiPartialSwapName,
      flag_descriptions::kUiPartialSwapDescription, kOsAll,
      SINGLE_DISABLE_VALUE_TYPE(switches::kUIDisablePartialSwap)},
+#if defined(OS_CHROMEOS) || defined(OS_ANDROID)
```

Comment:
Commit message: could we include the full commit message from https://chromium-review.googlesource.com/c/chromium/src/+/532294 / https://chromium.googlesource.com/chromium/src/+/e95d904578c265a84f117d7d5eff598423f9283d rather than just a link to it?

Reply:
Sure, I will update the PR including that message.